configs/fomalhaut.config

```diff
@@ -35,5 +35,5 @@ NVCCFLAG_FLU -Xptxas -dlcm=ca -prec-div=false -ftz=true
 NVCCFLAG_POT -Xptxas -dlcm=ca
 
 #gpu
-GPU_COMPUTE_CAPABILITY 860    # GeForce RTX 3080 and RTX A4000
-#GPU_COMPUTE_CAPABILITY 750    # GeForce RTX 2080 Ti
+# GPU_COMPUTE_CAPABILITY 860    # GeForce RTX 3080 and RTX A4000 (use CUDA 11)
+GPU_COMPUTE_CAPABILITY 750    # GeForce RTX 2080 Ti (default with CUDA 10)
```

Comment:
- Update `#gpu` to `# gpu`
- Remove the space before `GPU_COMPUTE_CAPABILITY 860`
- Switch the order of 750 and 860
- For the command, I prefer `CUDA 1X, g0x and g0x`

In short, I recommend the following setting
~~~
GPU_COMPUTE_CAPABILITY 750    # GeForce RTX 2080 Ti (CUDA 10, g01 and g02)
#GPU_COMPUTE_CAPABILITY 860    # RTX A4000 and GeForce RTX 3080 (CUDA 11, g03 and g04)
~~~